### ISSUE TITLE
 I don't know the address of the libc.memcpy_decision on zig-11-llvm-16 branch 

### ISSUE THREAD

**Issue by Anton-4** [roc-lang/roc#5861]
```
cargo test --locked --release -p roc_cli
```
```

---- cli_run::inspect_gui stdout ----
thread 'cli_run::inspect_gui' panicked at '
___________
The roc command:

  "D:\\a\\roc\\roc\\target\\release\\roc.exe build --optimize D:\\a\\roc\\roc\\examples\\inspect-gui.roc --"

had unexpected stderr:

  🔨 Rebuilding platform...
I don't know the address of the libc.memcpy_decision function! this may cause segfaults
I don't know the address of the libc.memcpy_decision function! this may cause segfaults

___________
', crates\cli\tests\cli_run.rs:148:13
stack backtrace:
   0: std::panicking::begin_panic_handler
             at /rustc/eb26296b556cef10fb713a38f3d16b9886080f26/library\std\src\panicking.rs:593
   1: core::panicking::panic_fmt
             at /rustc/eb26296b556cef10fb713a38f3d16b9886080f26/library\core\src\panicking.rs:67
   2: <log::NopLogger as log::Log>::log
   3: __ImageBase
   4: core::ops::function::FnOnce::call_once
             at /rustc/eb26296b556cef10fb713a38f3d16b9886080f26/library\core\src\ops\function.rs:250
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

failures:
    cli_run::breakout
    cli_run::breakout_hello_gui
    cli_run::hello_gui
    cli_run::inspect_gui
```

Discusion on zulip [here](https://roc.zulipchat.com/#narrow/stream/395097-compiler-development/topic/zig.200.2E11/near/392764631) and [here](https://roc.zulipchat.com/#narrow/stream/395097-compiler-development/topic/Windows.20Zig/near/390114487).
Only seen on windows x86_64.

**Comment by lukewilliamboswell** [comment]
Resolved in [d81c787](https://github.com/roc-lang/roc/pull/5851/commits/d81c787a6c1f21e7da329b27853c6355612f08ba)

**Comment by lukewilliamboswell** [comment]
Fixed by #5851

**Comment by Anton-4** [comment]
Thanks @lukewilliamboswell :tada:
